libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -58,6 +58,8 @@ WaveClip::WaveClip(
    bool copyCutlines)
     : mClipStretchRatio(orig.mClipStretchRatio)
     , mRawAudioTempo(orig.mRawAudioTempo)
+// Project tempo is not copied on purpose, since `orig` might come from another
+// project.
```

Comment:
It's good to see these comments.  Strictly they should be added in the commit "Project tempo changes affect WaveClip stretch ratios"

Reply:
But -- the comments are contradicted now by the changes in "Repair WaveTrack Copy/Paste/Join"

So what's right?

Did you try the cross-project case of copy/paste?

Did you copy copying to clipboard, closing source project, then pasting?

Reply:
Nice that you spot this, it's outdated. I later realized there were places where a track was copied for the clipboard, but before it was re-contextualized by being added to a project, boundary-related queries were made onto its clips. Hence the change of strategy: clip and track copies inherit the context from `orig`, and the pasting code is repsonsible for re-contextualization.
Will remove.